controllers/dellcsivolumegroupsnapshot_controller.go

```diff
@@ -1038,7 +1192,8 @@ func (r *DellCsiVolumeGroupSnapshotReconciler) HandleSnapContentDelete(obj inter
 
 			if snapshotsString == "" {
 				// All snapshotcontents have been deleted and we can delete the VG object
-				r.EventRecorder.Eventf(vg, v1.EventTypeWarning, common.EventReasonUpdated, "All managing snapshot contents are deleted")
+				//r.EventRecorder.Eventf(vg, v1.EventTypeWarning, common.EventReasonUpdated, "All managing snapshot contents are deleted")
```

Comment:
why leave this line commented out instead of deleting it?

Reply:
removed this commented line